src/Tools/Auditor/README.rst

```diff
@@ -0,0 +1,59 @@
+=============
+Dafny Auditor
+=============
+
+A tool to audit Dafny programs and report on assumptions contained
+within.
+
+Overview
+========
+
+It currently detects the following forms of assumption:
+
+* Anything marked with the ``{:axiom}`` attribute.
+
+* Declarations with at least one ``ensures`` clauses that
+
+  * lack a body, or
+
+  * are marked with the ``{:extern}`` attribute.
+
+* Explicit ``assume`` statements.
+
+In the future, it will detect additional constructs that introduce
+assumptions and integrate the results of other analyses.
+
+Building
+========
+
+From the top level of this repository, run the following.
+
+.. code-block:: shell
+
+  git clone https://github.com/dafny-lang/dafny
+  (cd dafny && git checkout compiler-bootstrap && make exe)
+  DAFNY_ROOT=$(pwd)/dafny make clean auditor
```

Comment:
```suggestion
   git clone https://github.com/dafny-lang/dafny
   (cd dafny && git checkout compiler-bootstrap && make exe)
   DAFNY_ROOT=$(pwd)/dafny make clean auditor
```

Reply:
You can use `git clone --branch compiler-bootstrap https://github.com/dafny-lang/dafny` to get the right branch

Reply:
But if we're going to have people clone that repo, won't they get a copy of this one from there?  Maybe not an up o date one…

Reply:
Yeah, it's slightly awkward, but I think it's the easiest sequence of steps to complete a build.